src/OrleansAzureUtils/Storage/AzureStorageUtils.cs

```diff
@@ -386,5 +389,38 @@ internal static string PrintStorageException(Exception exception)
                         (extendedError.AdditionalDetails != null && extendedError.AdditionalDetails.Count > 0) ?
                             String.Format(", ExtendedErrorInformation.AdditionalDetails = {0}", Utils.DictionaryToString(extendedError.AdditionalDetails)) : String.Empty);
         }
+
+        internal static JsonSerializerSettings ConfigureJsonSerializerSettings(IProviderConfiguration config)
+        {
+            // By default, use the standard Orleans serializer settings
+            var settings = OrleansJsonSerializer.SerializerSettings;
```

Comment:
We should clone the settings here, otherwise configuring it for 1 provider will affect all others that use JSON.